src/util.cpp

```diff
@@ -953,6 +953,20 @@ bool hasNonPublicParam(const CXXMethodDecl *decl)
     return false;
 }
 
+std::string getPythonBuiltinTypeSpelling(const std::string &type_name)
+{
+    if (type_name == "double" || type_name == "float")
+        return "float";
```

Comment:
Should we just do all the builtin types?  Most are pretty simple. :shrug: 
https://docs.microsoft.com/en-us/cpp/cpp/fundamental-types-cpp?view=vs-2019

Reply:
I think so. It won't be a 1to1 map though.